CONTRIBUTING.md

```diff
@@ -4,40 +4,39 @@ Contributions are always welcome, no matter how large or small. Before contribut
 please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Setup
-
 * Install Go 1.16
+* Install Docker 
 * Install [Soda CLI](https://gobuffalo.io/en/docs/db/toolbox)
-* Install Docker to run tests
-
-GoTrue uses the Go Modules support built into Go 1.11 to build. The easiest is to clone GoTrue in a directory outside of GOPATH, as in the following example:
-
-```sh
-$ git clone https://github.com/supabase/gotrue
-$ cd gotrue
-$ make deps
-```
-
-## Building
-
-```sh
-$ make build
-```
+  * `go install github.com/gobuffalo/pop/soda@latest`
+* Clone this repo: `git clone https://github.com/supabase/gotrue`
+* `cd gotrue`
+* To start the gotrue postgresql container running locally: `./hack/postgresd.sh` 
```

Comment:
`./hack/postgresd.sh` didn't work when we tried there error in the script from dbname I think @HarryET you remember ?

Reply:
hey @riderx, sorry for the late reply, thanks for pointing this out, are you able to recall the error message you encountered? 🤔